# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/MP4Interval.h
# Commit: aa102eb9990d
# Full Hash: aa102eb9990d6edbd5cf7473c5017082340764ff
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-05-24 21:42:08
# Regressor Bug: 1817997
# File Overlap Count: 3
# Description:
#   Bug 1817997 - Update the MP4 demuxer to use TimeUnit based on the internal MP4 time base. r=alwu
#   
#   This still gets the initial time value from mp4parse-rust, that is in
#   microseconds. mp4parse-rust has been updated to expose real time, and will be
#   updated later.
# ==============================================================================

diff -r 5c1b6ce71ca1 -r aa102eb9990d dom/media/mp4/MP4Interval.h
--- a/dom/media/mp4/MP4Interval.h	Wed May 24 13:18:40 2023 +0000
+++ b/dom/media/mp4/MP4Interval.h	Wed May 24 13:18:40 2023 +0000
@@ -12,7 +12,7 @@
 
 template <typename T>
 struct MP4Interval {
-  MP4Interval() : start(0), end(0) {}
+  MP4Interval() : start{}, end{} {}
   MP4Interval(T aStart, T aEnd) : start(aStart), end(aEnd) {
     MOZ_ASSERT(aStart <= aEnd);
   }